beta/src/pages/reference/usestate.md

```diff
@@ -68,17 +58,13 @@ function Form() {
 
 ---
 
-### `setState(nextState)` {/*setstate*/}
+### `set` functions, like `setName(nextState)` {/*setstate*/}
 
-`setState` lets you update the state to a different value and trigger a re-render:
+The `set` function returned by `useState` lets you update the state to a different value and trigger a re-render. You can pass the next state directly, or a function that calculates it from the previous state:
 
 ```js
-setState(nextState);
-```
-
-You can pass the next state directly, or a function that calculates it from the previous state:
+const [name, setName] = useState('Edward');
```

Comment:
👀

Reply:
You used Taylor for the initial state and the update so I had to make a name up!